public/css/caller_profile.css

```diff
@@ -429,6 +429,10 @@ aside .search-bar button {
     outline: none;
 }
 
+#user_profile_image{
```

Comment:
this ID doesn't exist in the html file

Reply:
Oh, thank you for pointing it out. Forgot to delete.